python/rpdk/java/templates/generate/HandlerWrapper.java

```diff
@@ -132,6 +132,7 @@ public JSONObject provideResourceSchemaJSONObject() {
             .desiredResourceState(requestData.getResourceProperties())
             .previousResourceState(requestData.getPreviousResourceProperties())
             .desiredResourceTags(getDesiredResourceTags(request))
+            .previousResourceTags(getPreviousResourceTags(request))
```

Comment:
This method is a new method. That mean anyone who updates the PyPi java plugin but without update the latest maven java plugin will run into problem, and it will break the build.

I'm afraid we need to copy the entire method `protected Map<String, String> getPreviousResourceTags(final HandlerRequest<ResourceT, CallbackT> request) {` into this HandlerWrapper as well as `replaceInMap`, which is a private method. 

It will be a little bit ugly. But comparing with breaking customers and wasting time to find and upgrade java plugin, I more prefer this temporary duplication.  

The entire `transform` method in the generated code seems not a good idea at all, but let's leave for further refactoring.

Reply:
Hmm, never mind, even we can duplicate this new method, but `getPreviousStackTags` still a new method and requires the Java plugin support.